vignettes/getting-started-nmbayes.Rmd

```diff
@@ -0,0 +1,343 @@
+path to your control stream _without the file extension_, in the same
+way you would create a standard NONMEM model with `bbr`, as described
+in the ["Getting Started with bbr and NONMEM"][gsb] vignette.  To tell
+`new_model()` to create a NONMEM Bayes model, pass "nmbayes" as its
+`.model_type` argument.
+
+```{r, include = FALSE}
+fs::file_copy(
+  file.path(REF_DIR, "bayes", "1100.ctl"),
+  file.path(MODEL_DIR, "1000.ctl")
+)
+```
+
+```{r}
+mod1000 <- new_model(file.path(MODEL_DIR, 1000), .model_type = "nmbayes")
+mod1000$model_type
+```
+
+## Creating a model object from an existing model
+
+While you can create a fresh NONMEM Bayes model with `new_model()`,
+more commonly you will create a new NONMEM Bayes model from an
+existing model, either a regular NONMEM model (`bbi_nonmem_model`) or
+a NONMEM Bayes model (`bbi_nmbayes_model`).
+
+### From a regular NONMEM model
+
+```{r, include = FALSE}
+copy_model_from(
+  read_model(
+    system.file(
+      "model", "nonmem", "basic", "1",
+      package = "bbr.bayes", mustWork = TRUE
+    )
+  ),
+  file.path(MODEL_DIR, "100")
+) %>% replace_all_tags(c("two-compartment + absorption", "FOCE"))
+```
+
+To create a NONMEM Bayes model from a regular NONMEM model, use
+`copy_model_as_nmbayes()`.
+
+```{r}
+mod100 <- read_model(file.path(MODEL_DIR, 100))
+mod1100 <- copy_model_as_nmbayes(mod100, 1100, .inherit_tags = TRUE) %>%
+  update_model_id() %>%
+  replace_tag("FOCE", "BAYES")
+
+mod100$model_type
+mod1100$model_type
+```
+
+That will copy over the control stream and YAML , like
+`copy_model_from()`, and then switch the model type to "nmbayes".  In
+addition, it will replace the estimation methods with two estimation
+records that serve as a template for defining a NONMEM Bayes run.
+
+```{r, echo = FALSE, message = FALSE, comment = ""}
+glue::glue_data_safe(
+  list(model_id = get_model_id(mod1100)),
+  bbr.bayes:::NMBAYES_HELP,
+  .trim = FALSE
+)
+```
+
+### From a NONMEM Bayes model
+
+To create a NONMEM Bayes from another one, use `copy_model_from()`,
+just as you would in `bbr` to [copy and iterate on a model][gsbc].
+
+```{r}
+mod1101 <- copy_model_from(mod1100) %>%
+  update_model_id()
+
+# Edit control stream...
+
+mod1101 <- add_notes(mod1101, "Switched from foo to bar")
+```
+
+# Submitting a model
+
+To execute a model, pass it to [`submit_model()`][].
+
+```{r, eval = FALSE}
+submit_model(mod1100)
+```
+
+Underneath the [`submit_model()`][] method of `bbi_nmbayes_model`
+objects creates several sub-models, one for generating the initial
+values and one for each MCMC chain.
+
+```{r, include = FALSE}
+fs::dir_copy(
+  file.path(REF_DIR, "bayes", "1100"),
+  file.path(MODEL_DIR, "1100"),
+  overwrite = TRUE
+)
+mod1100 <- read_model(file.path(MODEL_DIR, "1100"))
+```
+
+```{r}
+dir(get_output_dir(mod1100))
+```
+
+You shouldn't often need to directly access these sub-models because
+`bbr.bayes` implements custom methods and functions work with them
```

Comment:
missing "to" in "functions to works with them"

Reply:
Thanks for spotting.